### PR TITLE
IPFS Found

### PR DESCRIPTION
**Pull request by maobisheng**
Based on Filecoin network, we develop an application platform for storage and retrieval to help users store various types of files, including text, picture, audio, video, etc. [IPFS Found](https://filecoin.in) supports storage of multiple data sizes. We aim to build more decentralized, stable and robust data storage platform.

### CODE REVIEW

**Comment by dineshshenoy** [comment]
Thanks for submitting an application.  Our team will validate for completeness and then assign reviewers from the competitions review panel.

File: participants/IPFS-Found.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# <Project Name>
```
Comment (GregTheGreek) [inline participants/IPFS-Found.md:1]: This application has not been filled out.

Please answer the questions within the document to qualify.

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

**Comment by maobisheng** [comment]
@GregTheGreek @dineshshenoy please check again, I have add more infomation about my project-"IPFS Found"

File: participants/IPFS-Found.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## (3) What does your application/UI do?
+
+This is a completely data storage and retrieval platform，which supports the following features: 
+- Storage Client: user can easily storage the real data to the Filecoin network by "IPFS Found", including different types of data.
+- Retrieval Client: user can query the desired files by the file name or file CID.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+It will use many kinds of data from "curated" dataset.
```
Comment (dineshshenoy) [inline participants/IPFS-Found.md:29]: Can you be more specific here?

File: participants/IPFS-Found.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- Project Name : IPFS Found
+- Slack Handles : IPFSUnion.cn
+
+## (1) Brief Project Description
+
+Based on Filecoin network, we develop an application platform for storage and retrieval to help users store various types of files, including text, picture, audio, video, etc. IPFS-Found supports storage of multiple data sizes. We aim to build more decentralized, stable and robust data storage platform.
+
+## (2) Link to Project UI
+
+Link : https://filecoin.in
```
Comment (GregTheGreek) [inline participants/IPFS-Found.md:19]: Is this a usable website?

File: participants/IPFS-Found.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+We the data firstly will be retrieved from the local storage, if cannot, we will try to get it from the intermediate cache layer (such as IPFS). Finally, If it still doesn't work，we will take some time to have a retrieval deal from the Filecoin network.
+
+## (11) Who is the intended user for your application/UI?
+
+"IPFS Found" aim to support stable and robust storage and retrieval platform to everyone, but we will build it for individual users who need to store small amounts of data, and in the future enterprise clients will be considered.
+
+## (12) Do you have any users already or plans to acquire users soon?
+
+Yes. Our user-base is growing, and for now we have some clients from business partner. Besides, we are planning a strategy to attract some individual users.
```
Comment (GregTheGreek) [inline participants/IPFS-Found.md:61]: I wasn't able to use the listed website is this claim true?

**Review by GregTheGreek — DISMISSED** [review]

**Review by dineshshenoy — APPROVED** [review]

File: participants/IPFS-Found.md (hunk 2 above)
Comment (maobisheng) [reply participants/IPFS-Found.md:29]: We use "COVID-19 Open Research Dataset", "Chest X-Ray Images (Pneumonia)", "Huge Stock Market Dataset", "Open Images Dataset" and "Filecoin Proofs" in the Slingshot Competiton.

File: participants/IPFS-Found.md (hunk 3 above)
Comment (maobisheng) [reply participants/IPFS-Found.md:19]: We improve some tips and it can be used now.

File: participants/IPFS-Found.md (hunk 4 above)
Comment (maobisheng) [reply participants/IPFS-Found.md:61]: We didn't make it properly public before, but now it works. We're still improving.

**Review by cwaring — APPROVED** [review]

**Pull request merged**